billing/views.py

```diff
@@ -70,9 +101,13 @@ def invoice_payment_failed(self, invoice: stripe.Invoice) -> None:
         admins = get_all_admins_for_owners(owners)
 
         task_service = TaskService()
+        payment_intent = stripe.PaymentIntent.retrieve(
```

Comment:
Turns out you gotta go get the card in the PaymentIntent explicitly. Annoying that it's another fetch, but oh well, this code isn't time critical enough for that to matter.